PSKoans/Koans/Foundations/AboutHashtables.Koans.ps1

```diff
@@ -180,7 +180,7 @@ Describe 'Hashtables' {
             $Hashtable.ContainsValue('Fruit') | Should -BeTrue
 
             $Hashtable['Oranges'] | Should -Be 'Fruit'
-            $Hashtable['Carrots'] | Should -Not -Be $Hashtable['Oranges']
+            $Hashtable['Carrots'] | Should -Be $Hashtable['Oranges']
```

Comment:
This might need more clarification, for sure, but I don't think this is the problem here. 

The conditions as set without changing this line are that the hashtable must:

1. contain at least one key name that is `Carrots`
1. have at least one of the values be `'Fruit'`
1. the key `Oranges` should map to the value `'Fruit'`
1. the value for `Carrots` should not be the same as the value for `Oranges`

The intended solution is for carrots to be marked as `'Vegetables'`  when adding keys to the hashtable. 🙂 

Is there a way we can make that a bit more clear for this koan?